recipes/biogeme-optimization/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme_optimization-{{ version }}.tar.gz
+  sha256: a4e964d034208eb762217556112369d9f8f7dfeabe603a2dde360bca50031fca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=63.4.1
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - scipy >=1.14.0
+    - matplotlib-base >=3.9.0
+    - tomlkit >=0.12.5
+
+test:
+  imports:
+    - biogeme_optimization
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://biogeme.epfl.ch
+  summary: Various optimization algorithms for teaching and research
+  license: LicenseRef-BIOGEME-OPTIMIZATION
+  license_file: LICENSE
```

```suggestion
  license_file: LICENSE
  dev_url: https://github.com/michelbierlaire/optimization
```

This one took a little while to find 😀, so listing it explicitly would be helpful.

---

I have added this dev_url